phpcs.xml

```diff
@@ -33,8 +33,12 @@
 	<rule ref="SlevomatCodingStandard.ControlStructures.DisallowEqualOperators"/>
 	<rule ref="SlevomatCodingStandard.ControlStructures.RequireNullCoalesceOperator"/>
 	<rule ref="SlevomatCodingStandard.ControlStructures.EarlyExit"/>
+	<rule ref="SlevomatCodingStandard.ControlStructures.UselessConditionWithReturn"/>
```

Comment:
I see that some sniffs from https://github.com/slevomat/coding-standard/releases/tag/4.8.0 are missing here, can you add them? I always evaluate each sniff, if it doesn't cause much problems, I add it.

Reply:
@ondrejmirtes I'll add those I haven't added yet and ping you :)